src/hotspot/share/gc/serial/defNewGeneration.hpp

```diff
@@ -50,6 +50,7 @@ class STWGCTimer;
 
 class DefNewGeneration: public Generation {
   friend class VMStructs;
+  friend class SerialHeap;
```

Comment:
Not really know whether it is a good way. Maybe we can export  a method `DefNewGeneration::gc_timer`.